lib/schemaUtils.js

```diff
@@ -198,6 +199,85 @@ module.exports = {
 
   safeSchemaFaker: safeSchemaFaker,
 
+  analyzeSpec: function (spec) {
+    var size,
+      numberOfRefs = 0,
+      specString,
+      numberOfRequests = 0;
+
+    // Stringify and add whitespaces as there would be in a normal file
+    // To get accurate disk size
+    specString = JSON.stringify(spec, null, 2);
```

Comment:
why null, 2? That'll add whitespace, and isn't what we want to measure.

Reply:
Okay 👍

Reply:
@umeshp7 ?